circle.yml

```diff
@@ -38,42 +35,98 @@ defaults:
         cd build
         cmake ..
         cmake --build . -- -j4
+
   vm-tests: &vm-tests
     run:
       name: "Run ethereum VM tests"
       command: |
         npm run vmTests
+
+  clone-cpp-eth: &clone-cpp-eth
+    run:
+      name: "Clone cpp-ethereum and hera"
+      working_directory: ~/
+      # to fetch Hera branch that does evm2wasm.js --trace:
+      # cd hera
+      # git fetch origin evm2wasm-trace
+      # git checkout evm2wasm-trace
+      command: |
+        git clone https://github.com/ethereum/cpp-ethereum --branch ewasm-json-trace --single-branch
+        cd cpp-ethereum
+        git reset --hard b85a810d5b003d3d5046c0664d3d49e9bfe58939
+        git submodule update --init
+        cd hera
+        git pull origin master
+
+  build-cpp-eth: &build-cpp-eth
+    run:
+      name: "Build cpp-ethereum and hera"
+      working_directory: ~/
+      command: |
+        mkdir -p build
+        cd build
+        cmake ../cpp-ethereum -DCMAKE_BUILD_TYPE=RelWithDebInfo -DHERA=ON -DHERA_DEBUGGING=ON -DHERA_EVM2WASM=ON -DHERA_EVM2WASM_EVM_TRACE=ON
```

Comment:
I changed `-DHERA_EVM2WASM_EVM_TRACE` to be a runtime option for Hera: https://github.com/ewasm/hera/pull/174 .